src/_gettsim/parameters/ges_rente.yaml

```diff
@@ -975,6 +976,60 @@ abolishment_cohort_rente_für_arbeitsl:
   note: https://www.sozialgesetzbuch-sgb.de/sgbvi/237.html
   1980-01-01:
     scalar: 1952
+altersgrenze_arbeitsl_abschlagsfrei_vertrauensschutz:
+  name:
+    de: >-
+      Altersgrenze für Altersrente wegen Arbeitslosigkeit mit Vertrauensschutz
+      (abschlagsfrei).
+    en: >-
+      Age threshold for pension for unemployed (without deductions)
+      with Vertrauensschutz
+  description:
+    de: >-
+      Frühestmögliches Alter ab dem die Altersrente für Arbeitslose, für die der
+      Vertrauensschutz gilt, abschlagsfrei bezogen werden kann. Unter bestimmten
+      Veraussetzungen sind Personen, die die Bedingungen für die Rente für
+      Arbeitslose erfüllen, von der Anhebung der Altersgrenzen zu frühzeitigen
+      Inanspruchnahme nicht betroffen bzw. es erfolgt eine geringere Anhebung.
+      (Z.B. wenn bereits eine Arbeitslosigkeit vorgelegen hat, als das Gesetz
+      beschlossen wurde.)
+    en: >-
+      Earliest possible age to receive pension for unemployed without deductions
+      under legitimate expectations.
+  reference: § 237 SGB VI Abs. 4
+  note: >-
+    https://www.gesetze-im-internet.de/sgb_6/__237.html
+  2005-12-31:
```

Comment:
These should become months 1-12 instead of 1, 2, 3

Why is the starting date 2005-01-01 but 2005-12-31 for the next entry? Should be only one, I  suppose?